CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [BUG][Fuctional Test] Make setDefaultAbsoluteRange more robust and update doc views tests ([#5242](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5242))
 - [BUG] Add platform "darwin-arm64" to unit test ([#5290](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5290))
 - [BUG][Dev Tool] Add dev tool documentation link to dev tool's help menu [#5166](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5166)
+- [BUG] Improve the display content of the error toasts ([#5345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5345))
```

Comment:
This is an enhancement; isn't it? Also, we don't need to add `[BUG]` for bug fixes because they are listed in a section meant for bug fixes.